Part-1-Functions

```diff
@@ -21,7 +21,13 @@
 # arrayCheck([1, 1, 2, 1, 2, 3]) → True
 
 def arrayCheck(nums):
-    # CODE GOES HERE
+    num=[1,3,5,7]
+    for i in num:
```

Comment:
The Program is wrong please recheck the error
Don't use num the function already has a parameter nums  
and Don't use num for For Loop.
There is a index out of bound error if you run this code.